tests/test_utils.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+"""Tests for coronawhy_vt utilities."""
+
+import unittest
+from math import inf
+
+# Project specific packages
+from coronawhy_vt.utils.local_processing import make_ngram_map_and_list,\
```

Comment:
if you put parentheses around the imports, you don't need ugly line breaks. A modern IDE like PyCharm should automatically reformat this stuff for you